### PR TITLE
Hotfix fu 20 remove name validation

### PR DESCRIPTION
**Pull request by SooLiD**

### CODE REVIEW

File: furms-web-ui/src/main/java/io/imunity/furms/ui/views/site/settings/SettingsView.java [hunk 1]
```diff
@@ -169,6 +172,12 @@ private void doSaveAction(Binder<SiteSettingsDto> binder) {
...
 				refreshBinder(binder);
 				showSuccessNotification(getTranslation("view.site-admin.settings.form.button.save.success"));
+			} catch (DuplicatedNameValidationError e) {
+				findFormField(binder, TextField.class, NAME_FIELD_ID)
+						.ifPresent(textField -> {
+							textField.setErrorMessage(getTranslation("view.site-admin.settings.form.name.validation.unique"));
+							textField.setInvalid(true);
+						});
 			} catch (RuntimeException exception) {
 				showErrorNotification(getTranslation("view.site-admin.settings.form.error.unexpected"));
```
Comment (golbi) [inline furms-web-ui/src/main/java/io/imunity/furms/ui/views/site/settings/SettingsView.java:182]: Exception details are lost and not logged

File: furms-web-ui/src/main/java/io/imunity/furms/ui/views/site/settings/SettingsView.java [hunk 2]
```diff
@@ -169,6 +172,12 @@ private void doSaveAction(Binder<SiteSettingsDto> binder) {
 						.build());
 				refreshBinder(binder);
 				showSuccessNotification(getTranslation("view.site-admin.settings.form.button.save.success"));
+			} catch (DuplicatedNameValidationError e) {
+				findFormField(binder, TextField.class, NAME_FIELD_ID)
```
Comment (golbi) [inline furms-web-ui/src/main/java/io/imunity/furms/ui/views/site/settings/SettingsView.java:176]: Why to search for a name field if you know that you want the name field? that's makes no sense :-) just use name.

File: furms-web-ui/src/main/java/io/imunity/furms/ui/utils/FormUtils.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.lang.String.format;
+import static java.util.stream.Collectors.toList;
+
+public class FormUtils {
+
+	public static <T extends Component> Optional<T> findFormField(Binder<SiteSettingsDto> binder, Class<T> fieldType, String id) {
```
Comment (golbi) [inline furms-web-ui/src/main/java/io/imunity/furms/ui/utils/FormUtils.java:20]: This code is specific to sites currently. But to be dropped completely - see below.

**Review by golbi — CHANGES_REQUESTED** [review]

File: furms-web-ui/src/main/java/io/imunity/furms/ui/views/site/settings/SettingsView.java (hunk 1 above)
Comment (rkrysinski) [reply furms-web-ui/src/main/java/io/imunity/furms/ui/views/site/settings/SettingsView.java:182]: +1

**Review by rkrysinski — CHANGES_REQUESTED** [review]

**Pull request merged**